docs/en/ingest-management/elastic-agent/advanced-kubernetes-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,104 @@
+[[advanced-kubernetes-managed-by-fleet]]
+= Advanced {agent} configuration managed by {fleet}
+
+:manifest: https://raw.githubusercontent.com/elastic/elastic-agent/{branch}/deploy/kubernetes/elastic-agent-managed-kubernetes.yaml
+
+For basic {agent} managed by {fleet} scenarios follow the steps in <<running-on-kubernetes-managed-by-fleet>>.
+
+On managed {agent} installations it can be useful to provide the ability to configure more advanced options, such as the configuration of providers during startup. Refer to <<providers>> for more details.
+
+Following steps demonstrate above scenario:
+
+[discrete]
+== Step 1: Download the {agent} manifest
+
+include::run-container-common/download-elastic-agent.asciidoc[]
+
+[discrete]
+== Step 2: Create a new configmap
+
+[source,yaml]
+.Create a new configmap
+------------------------------------------------
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: agent-node-datastreams
+  namespace: kube-system
+  labels:
+    k8s-app: elastic-agent
+data:
+  agent.yml: |-
+    providers.kubernetes_leaderelection.enabled: false
+    fleet.enabled: true
+    fleet.access_token: "<FLEET_ENROLLMENT_TOKEN>"
+---
+------------------------------------------------
+
+Notes:: 
+1. The above example demonstrates how to disable the `kubernetes_leaderelection` provider. For alternative scenarios you can follow the same procedure. For example, the following settings configure the `kubernetes` provider metadata enrichment:
+
+[source,yaml]
+.Example of configmap for kubernetes provider configuration
+------------------------------------------------
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: agent-node-datastreams
+  namespace: kube-system
+  labels:
+    k8s-app: elastic-agent
+data:
+  agent.yml: |-
+    providers.kubernetes: 
+      add_resource_metadata: 
+        deployment: true
+        cronjob: true
+    fleet.enabled: true
+    fleet.access_token: "<FLEET_ENROLLMENT_TOKEN>"
+---
+------------------------------------------------
+
+Notes:: 
+2. Find more information about https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html[Enrollment Tokens].
+
+[discrete]
+== Step 3: Configure Daemonset
+
+Inside the downloaded manifest, update the Daemonset resource:
+
+[source,yaml]
+.Update entrypoint 
+------------------------------------------------
+containers:
+  - name: elastic-agent
+    image: docker.elastic.co/beats/elastic-agent: <Image Version>     #< Change this with the image version of agent
```

Comment:
should we maybe rather refer to the copied manifest from the Add agent on K8s instead of the raw manifest? To avoid such comments about changing version?
I am about this page:
<img width="908" alt="Screenshot 2024-01-12 at 15 01 42" src="https://github.com/elastic/ingest-docs/assets/28299531/3c05e59c-d1cf-4da5-a560-471a7715daad">

Reply:
I lked your idea @tetianakravchenko , I updated step 1 with the manifest to read it from UI and here just added a comment 

Let me know if ok